### PR TITLE
Implemented FT_76, FT_80

### PR DESCRIPTION
**Pull request by AnkitGaurNms**
Implemented test cases FT_76 and FT_80

### CODE REVIEW

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by frankhuster** [comment]
ok to test

**Comment by frankhuster** [comment]
lgtm

**Comment by frankhuster** [comment]
@AnkitGaurNms, you need to merge master into this branch and push again, @srivastavaAbhi can talk you through it, if needed.

**Comment by AnkitGaurNms** [comment]
Merged branch 'master' into this branch.

File: testing/src/test/java/org/motechproject/nms/testing/it/api/KilkariControllerBundleIT.java [hunk 1]
```diff
@@ -689,4 +688,104 @@ public void verifyFT77()
...
     }
 
+    @Test
+    public void verifyFT76() throws IOException, InterruptedException {
+        /*
+         * To check that only message for Active Pack should be returned from
+         * inbox after 7 days of user's subscription gets completed for 72Weeks
+         * Pack while user is subscribed for both Pack.
+         */
+        setupData();
```
Comment (frankhuster) [inline testing/src/test/java/org/motechproject/nms/testing/it/api/KilkariControllerBundleIT.java]: setupData() is called automatically on line 113 before each test, it's not needed here.

File: testing/src/test/java/org/motechproject/nms/testing/it/api/KilkariControllerBundleIT.java [hunk 2]
```diff
@@ -689,4 +688,104 @@ public void verifyFT77()
...
+    }
+
+    @Test
+    public void verifyFT80() throws IOException, InterruptedException {
+        /*
+         * To check that only message for Active Pack should be returned from
+         * inbox after 7 days of user's subscription gets completed for 48Weeks
+         * Pack while user is subscribed for both Pack.
+         */
+        setupData();
```
Comment (frankhuster) [inline testing/src/test/java/org/motechproject/nms/testing/it/api/KilkariControllerBundleIT.java]: setupData() is called automatically on line 113 before each test, it's not needed here.

**Comment by frankhuster** [comment]
Thanks for merging @AnkitGaurNms, there are a couple of changes needed.

**Comment by frankhuster** [comment]
ok to test

**Comment by frankhuster** [comment]
ok to test

**Pull request merged**